src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,67 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
+"""
+Jinja support.
+"""
+import typing as t
```

Comment:
Import whatever you need `from typing import ...` instead of abstract import renaming.